giraffe/src/utils/legend/sort.test.ts

```diff
@@ -0,0 +1,81 @@
+import {getDataSortOrder} from './sort'
+import {getRandomTable} from '../randomTable'
+import {lineTransform} from '../../transforms/line'
+import {NINETEEN_EIGHTY_FOUR} from '../../constants/colorSchemes'
+import {DomainLabel} from '../../types'
+
+describe('getDataSortOrder', () => {
+  const xColKey = '_time'
+  const yColKey = '_value'
+  const maxValue = 100
+  const numberOfRecords = 200
+  const recordsPerLine = 10
+  const fillColKeys = ['cpu', 'host', 'machine']
+  const table = getRandomTable(
+    maxValue,
+    true,
+    numberOfRecords,
+    recordsPerLine,
+    fillColKeys
+  )
+
+  it('overlaid line graphs should be unsorted', () => {
```

Comment:
it's a little more work, but can you try to convert these into more naturally flowing language? the `it` of the method call is supposed to be read as part of the sentence:

>it overlaid line graphs should be unsorted

might be converted to:

>it leaves overlaid line graphs unsorted

```ts
it('leaves overlaid line graphs unsorted', () => {}
```

it's kind of nitpicky, but it's rspec / jasmine convention, and writing them this way shows up when printing the specs in a report. if you run `jest` with `--verbose` you can see what I'm talking about. Let's just compare two random test suites:

```
 PASS  src/utils/isFiniteNumber.test.ts
  isFiniteNumber
    ✓ the value created from the Number constructor is a finite number (29 ms)
    ✓ the value created from the Number function is a finite number (3 ms)
    ✓ a literal number and constants are finite numbers
    ✓ Infinity and negative Infinity are not finite numbers (1 ms)
    ✓ the value NaN is not a finite number
    ✓ anything else is not a finite number (1 ms)

 PASS  src/utils/replace.test.ts
  replace
    ✓ replaces only the first match of the second argument string with the third argument
    ✓ handles the second argument RegEx with correct replacement using the third argument
    returns the first argument when given less than 3 arguments
      ✓ returns "undefined" as a string when first argument is undefined (1 ms)
      ✓ returns "null" as a string when first argument is null
      ✓ returns the first argument as a string (1 ms)
```

like I said, minor thing, but look how `replace` flows like a sentence, but `isFinite` is kind of disjointed. if all of our tests flow more naturally, we can actually print out our tests in human-readable language, which might be very helpful for not much more effort.

Reply:
Oh yeah, definitely. I did that very quickly. Will change the language.